docs/service-patterns.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layouts/service-patterns.njk
+title: Adding service patterns to the MoJ design system
+---
+
+Service patterns help you to determine the steps and tasks necessary to design your service. You can also use service patterns to scope your work or identify levels of complexity.
+
+Any action or activity a user completes is a potential service pattern, like booking appointments, making referrals, or tracking progress.
+
+## What MoJ service patterns might look like
+
+We think service patterns should be:
+
+- based on common tasks that users complete as part of a service
+- reusable to encourage consistent experiences
+- modular so that they can be combined to create complete services
+- designed for multiple channels like offline services
+
+Each service pattern entry will contain guidance on how to implement the service pattern in a consistent way across our services.
+
+Service pattern information might include things like:
+
+- examples of implementation
+- customer journey maps
+- scenario mapping
+- supporting research
+- accessibility considerations
+- relevant components and patterns
+
+Service patterns will not replace design. Instead, service patterns will be a starting point to accelerate design and encourage consistent experiences.
+
+Each pattern must be unique and useful so that multiple products can use it.
+
+## Suggested MoJ service patterns
+
+We are developing our first service pattern right now.
+
+MoJ designers have already identified around 25 potential service patterns, such as:
+
+- book, view, and manage appointments
+- apply for something and track progress
+- order and pay for something
+- assign and complete activities
+
+This is not a complete list, and we may have missed something, but it's a good starting point.
+
+## Contributing to MoJ service patterns
+
+You will likely design multiple instances of potential service patterns when you develop new products or services.
+
+Speak to <a href="mailto:martin.ford-downes@digital.justice.gov.uk">Martin Ford-Downes</a> if you have a potential service pattern idea or want to help us work on others.
```

Comment:
Martin is the single point of contact because he's leading on Service Patterns. He's also the one that will be doing the promoting. This is very much an internal objective, not a x-gov thing (for now), hence the page has no link from anywhere on the site.

If service patterns is a thing that grows, it'll become more embedded into the design system and the design system team, where there'll no longer be a single point of contact.

I did check and Martin was happy for his email address to be used.